Tests/Remora.Discord.Commands.Tests/ContextTests.cs

```diff
@@ -52,7 +53,7 @@ public class ContextTests
     public ContextTests()
     {
         _services = new ServiceCollection()
-            .AddDiscordRest(_ => "dummy")
+            .AddDiscordRest(_ => ("dummy", DiscordTokenType.Bearer))
```

Comment:
Slightly strange to use Bearer here, since Bot is the "standard" option. The use of Bearer implies that it has some effect on the test, which I don't believe is correct.